crates/sound/src/device.rs

```diff
@@ -566,7 +561,7 @@ impl VhostUserSoundBackend {
             threads,
             virtio_cfg: VirtioSoundConfig {
                 jacks: 0.into(),
-                streams: 1.into(),
+                streams: Le32::from(streams_no as u32),
```

Comment:
Is this related?

Reply:
Because RX is implemented, the second stream that is already allocated above as input won't be visible to the guest if the `streams` count is 1